### PR TITLE
Up19 104 reimplement login

### PR DESCRIPTION
**Pull request by murphyad**
Implemented a new site login page using Semantic UI Framework.

- Page notifies user if login is unsuccessful
- Page redirects user on successful login to site home page
- User's token is stored in the redux global store
- Page does not have a link to user registration page because that page and route is not currently on this branch.

### CODE REVIEW

**Comment by bhaugen** [comment]
Is https://semantic-ui.com/ something y'all are using more generally, or just for this page?

(Or did I misunderstand and that is not even the correct URL...)

**Comment by larsonme** [comment]
@bhaugen , Yes, we are planning on using it in general for new development.  We noticed that we were spending a large amount of development time on making elements look "nice" which slowed our progress alot.  For reference, we are using the react version of semantic found at https://react.semantic-ui.com/

File: src/pages/LoginPage.js [hunk 1]
```diff
@@ -31,37 +41,64 @@ class LoginPage extends React.Component {
         this.props.createToken({variables: mutationVariables}).then((response) => {
             let token = response.data.createToken.token;
             this.props.currentUserActions.setCurrentUserToken(token);
-            if(this.props.currentUserToken !== "N/A"){
+            if (this.props.currentUserToken !== "N/A") {
+                this.setState({loginFailed: false});
                 // DISPLAY SUCCESS / REDIRECT
+                this.setState({loginSuccess: true});
             }
             // console.log(token);
```
Comment (larsonme) [inline src/pages/LoginPage.js]: Remove this commented out console.log statement

File: src/pages/LoginPage.js [hunk 2]
```diff
@@ -1,18 +1,28 @@
 import React from 'react';
-import {Form, Button} from 'semantic-ui-react'
+import {Form, Button, Grid, Header, Segment, Message} from 'semantic-ui-react'
 import 'semantic-ui-css/semantic.min.css'
 import createToken from "../queries/createToken.js";
 import * as currentUserActions from '../redux/actions/currentUserActions';
 import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
+import { withRouter, Redirect} from 'react-router-dom';
+//import {COLOR_BUTTON_SUBMIT} from "../resources/colors.js";
```
Comment (larsonme) [inline src/pages/LoginPage.js]: remove this unused inport

**Review by larsonme — APPROVED** [review]
Just a couple comments about removing some stuff.  Otherwise, good work @murphyad !

File: src/pages/LoginPage.js [hunk 3]
```diff
@@ -31,37 +41,64 @@ class LoginPage extends React.Component {
...
             this.props.currentUserActions.setCurrentUserToken(token);
-            if(this.props.currentUserToken !== "N/A"){
+            if (this.props.currentUserToken !== "N/A") {
+                this.setState({loginFailed: false});
                 // DISPLAY SUCCESS / REDIRECT
+                this.setState({loginSuccess: true});
             }
             // console.log(token);
         }).catch((error) => {
             console.log("Caught an error");
```
Comment (StrikeNP) [inline src/pages/LoginPage.js]: This error message should be more  descriptive. In the console logs, it's not guaranteed to appear next to the error printout given below.

**Review by StrikeNP — APPROVED** [review]
I recommend fixing a console log I noted, but it's not severe enough to prevent a merge.

**Review by DonalMoloneyMSOE — APPROVED** [review]

**Pull request merged**